en/docs/learn/configuring-openid-connect-back-channel-logout.md

```diff
@@ -31,13 +31,31 @@ back-channel logout.
 4.  Upon receiving the logout token, the client application validates
     the logout token and invalidates the user session.
 
+
+### Configuring the sample applications
+
+1. Follow the steps
+   [here](../../develop/deploying-the-sample-app/#deploying-playground2-webapp)
+   to download, deploy and register `playground2` applications.
+
+
+2.  Make a copy of ` playground2.war `it and rename to `
+    playground3.war` in the same location described in step 1.
```

Comment:
shall we remove the "it" in this sentence:

 "Make a copy of playground2.war **it** and rename to...."

Reply:
Fixed